packages/react-button/src/components/CompoundButton/useCompoundButtonClasses.ts

```diff
@@ -43,7 +48,12 @@ export const useCompoundButtonClasses = makeClasses<CompoundButtonState>({
     {
       color: 'var(--button-secondaryContentColor)',
       fontSize: 'var(--button-secondaryContentFontSize)',
+      fontWeight: 'var(--button-secondaryContentFontWeight)',
       lineHeight: '100%',
+
+      '&:nth-child(2)': {
```

Comment:
i think we do `:not(:first-child)` in other places; i think that might be a little better, as it will apply to additional items if we add later. I'm not sure about perf diff.